relays/bin-substrate/src/cli/bridge.rs

```diff
@@ -155,6 +70,20 @@ pub trait CliBridge: CliBridgeBase {
 	>;
 }
 
+pub trait MessagesCliBridge: CliBridgeBase {
+	const ESTIMATE_MESSAGE_FEE_METHOD: &'static str;
+	type MessagesLane: SubstrateMessageLane<
+		SourceChain = Self::Source,
+		TargetChain = Self::Target,
+		SourceTransactionSignScheme = Self::Source,
+		TargetTransactionSignScheme = Self::Target,
+	>;
+
+	fn derive_account_from_id(
```

Comment:
This actually shall be removed in the future (there's an issue for that somewhere in the repo). But of course, not it this PR - just sharing the info :)